src/Spectre.libGenetic.Tests/IndividualTest.cpp

```diff
@@ -26,13 +26,24 @@ namespace
 {
 using namespace spectre::algorithm::genetic;
 
+const auto seed = 0ul;
+
 TEST(IndividualInitialization, initializes)
 {
     EXPECT_NO_THROW(Individual({ true, true, true, true }));
     EXPECT_NO_THROW(Individual({ false, false, false, false }));
     EXPECT_NO_THROW(Individual({ true, false, true, false }));
 }
 
+TEST(IndividualInitialization, initializes_by_shuffle)
```

Comment:
Similarly, as in the case of `Generation`. We can provide single factory for random `Generation`, which creates random `Individual`s and assembles them into a `Generation`.

Reply:
done